modules/ossm-cert-manager-integration-istio.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: CONCEPT
+[id="ossm-cert-manager-integration-istio_{context}"]
+== Integrating Service Mesh with cert-manager and istio-csr
+
+cert-manager adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates. 
+
+It obtains certificates from a variety of Issuers, both popular public Issuers as well as private Issuers, and ensures the certificates are valid and up-to-date. It also attempts to renew certificates at a configured time before expiry. 
+
+cert-manager enables Istio users to integrate with their private or external PKI. `istio-csr` acts as an external CA for Istio. When it is enabled, applications within a mesh send requests for signing certificates to `istio-csr`. 
+
+.Prerequisites
+
+* cert-manager v1.11+ or community cert-manager operator v1.11+ or cert-manager operator provided by Red Hat v1.10+.
```

Comment:
I find it a little hard to read this line with "...  or .. or ... ", perhaps it'd read better as a nested bulleted list.

Reply:
Good point. Thank you!